pkg/version/version.go

```diff
@@ -23,6 +23,9 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// NOTE: use go build -ldflags "-X github.com/tektoncd/cli/pkg/version.defaultNamespace=tekton-pipelines"
+var defaultNamespace = "tekton-pipelines"
```

Comment:
Is not better to put defaultNamespace value in flag itself

Reply:
@piyush-garg wdym ? we need to have that variable "defined" so that we can overwrite it with the `ldflags`. Being here seems to make more sense to me than in the `cmd` part (with the flags).

Reply:
Replied here https://github.com/tektoncd/cli/pull/1092/files#r464241241